test/query_test.rb

```diff
@@ -363,6 +378,22 @@ def test_modulo
     assert_equal 2, client.query { modulo(2) }
   end
 
+  def test_lt
+    assert_equal true, client.query { lt 1, 2 }
```

Comment:
(At least, behavior doesn't matter unless the client makes some assumption about it.)